github/admin.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "fmt"
+
+// AdminService handles communication with the admin related methods of the
+// GitHub API. These API routes are normally only accessible for GitHub
+// Enterprise installations.
+//
+// GitHub API docs: https://developer.github.com/v3/enterprise/
```

Comment:
Should this URL be https://developer.github.com/v3/enterprise/ldap/? That seems to be the specific page for the 2 endpoints being supported by `AdminService` here.

Unless the plan is to expand `AdminService` to cover all other parts of https://developer.github.com/v3/enterprise/, including https://developer.github.com/v3/enterprise/admin_stats/, https://developer.github.com/v3/enterprise/license/, https://developer.github.com/v3/enterprise/management_console/, etc.?